setup.py

```diff
@@ -21,7 +21,7 @@
 import setuptools  # type: ignore
 
 name = "google-cloud-aiplatform"
-version = "0.3.0"
+version = "0.3.1"
```

Comment:
No need to bump this here - if you merge this as a `feat` you'll see a PR for `0.4.0`.